src/button_handler.cpp

```diff
@@ -292,6 +347,14 @@ void Handler::powerBtnPressed(sdbusplus::message::message& msg)
 {
     try
     {
+        if (!isBmcReady())
```

Comment:
This should not be needed here since we only want to handle the case where we are powered OFF, and that will get checked anyway when you call powerPressed from here.

Reply:
👍